src/document.h

```diff
@@ -1205,6 +1205,23 @@ struct Document {
                 return scaledviewingmode ? _(L"Now viewing TreeSheet to fit to the screen exactly, press F12 to return to normal.")
                                          : _(L"1:1 scale restored.");
 
+            case A_FILTERRANGE: {
+                wxString beginrange_str = wxGetTextFromUser(
+                    _(L"Please type in the start datetime. Both date AND time must be entered."), 
+                    _(L"Start datetime"), "1970-01-01T12:00:00", sys->frame);

```

Comment:
I revised the commit. Is it better now? It is a little bit difficult to place the control elements because the sizes are handled differently on each platform, even between X11 and Wayland there is a difference.